tools/cli/README.md

```diff
@@ -0,0 +1,18 @@
+# dubbo-go-cli
+
+### 1. 解决问题
```

Comment:
hey, guy, pls using English in an Apache project. Or u rename this file to readme_cn.md and add an English readme.